src/components/NcDashboardWidget/NcDashboardWidget.vue

```diff
@@ -375,4 +365,9 @@ export default {
 		}
 	}
 }
+
+::v-deep .empty-content {
```

Comment:
Migrate to use `:deep` selector instead of deprecated `::v-deep` https://github.com/nextcloud/nextcloud-vue/pull/3348

Reply:
Weird, should have been caught by CI

Reply:
Ah, the rule is not in this repo yet

Reply:
Still need to be change :)